src/main/kotlin/ch/komunumo/server/business/configuration/control/ConfigurationService.kt

```diff
@@ -33,13 +36,42 @@ object ConfigurationService {
 
     private fun loadConfiguration(): Properties {
         val properties = Properties()
-        val file = Paths.get(System.getProperty("user.home"), ".komunumo", configurationFilename).toFile()
+        val filePath = System.getenv(envVariableName) ?: ""
+        val file: File = configFile(filePath)
```

Comment:
I had to read these lines twice to understand, what they do. Maybe it is better to extract that into a private function `buildConfigurationFilename` or similar.

Reply:
Alternative: I move the filePath into the configFile routine and rename configFile into buildConfigFile?